storage.go

```diff
@@ -22,6 +22,35 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 	return o
 }
 
+func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
+	rp := s.getAbsPath(path)
+
+	var nextPos int64 = 0
+	isExist, err := s.bucket.IsObjectExist(rp)
```

Comment:
We don't need to check if this object exists or not, just return error that we got.